genet/use/recipes.py

```diff
@@ -0,0 +1,889 @@
+import json
+import logging
+import os
+import time
+from pathlib import Path
+from typing import Any, Optional
+
+import geopandas as gpd
+import pandas as pd
+from pyproj import CRS
+
+import genet as gn
+import genet.output.sanitiser as sanitiser
+from genet.utils.persistence import ensure_dir
+from genet.variables import EPSG4326
+
+
+def _to_json(dict_to_save: dict, filepath: Path) -> None:
```

Comment:
This could(/should?) be in `output` or an `io.py` util file

Reply:
e.g., `output/json.py` Doesn't matter if it's just one method I'd say

Reply:
I like the idea of moving IO functions into an `io` module. It's a natural separation.